This patch removes protection from locale files installed by XaoS, since their
protection interferes with the normal operation of GSRC. If you're installing 
XaoS system-wide using GSRC and you have root priveleges, you can disable this
patch.

diff -x config.log -x config.status -ru tmp/xaos-3.5/src/i18n/Makefile.in.in work/xaos-3.5/src/i18n/Makefile.in.in
--- work/xaos-3.5/src/i18n/Makefile.in.in	2009-07-18 01:50:28.000000000 +0200
+++ work/xaos-3.5/src/i18n/Makefile.in.in	2013-01-15 23:49:47.827560898 +0100
@@ -2,7 +2,7 @@
 LANGUAGES=$(PO-FILES:.po=)
 
 # Maybe this should be autogenerated by "configure".
-LOCALEDIR=@LOCALEDIR@
+LOCALEDIR=$(prefix)/share/locale
 
 # If other files will be translated, they must be also here:
 CSOURCES=../ui/ui.c ../ui-hlp/menu.c \
@@ -21,13 +21,13 @@
 install: $(MO-FILES)
 	@for i in $(LANGUAGES); do \
 	 @INSTALL@ -d $(DESTDIR)$(LOCALEDIR)/$$i/LC_MESSAGES; \
-	 @INSTALL@ -m 444 $$i.mo $(DESTDIR)$(LOCALEDIR)/$$i/LC_MESSAGES/xaos.mo; \
+	 @INSTALL@ -m 666 $$i.mo $(DESTDIR)$(LOCALEDIR)/$$i/LC_MESSAGES/xaos.mo; \
 	 done
 
 install-win: $(MO-FILES)
 	@for i in $(LANGUAGES); do \
 	 @INSTALL@ -d $(DESTDIR)/locale/$$i/LC_MESSAGES; \
-	 @INSTALL@ -m 444 $$i.mo $(DESTDIR)/locale/$$i/LC_MESSAGES/xaos.mo; \
+	 @INSTALL@ -m 666 $$i.mo $(DESTDIR)/locale/$$i/LC_MESSAGES/xaos.mo; \
 	 done
 
 clean:
